internal/pkg/term/cursor/cursor.go

```diff
@@ -88,3 +88,10 @@ func (c *Cursor) EraseLine() {
 		terminal.EraseLine(cur.Out, terminal.ERASE_LINE_ALL)
 	}
 }
+
+// EraseLine erases a line if the writer w is a file, like Stdout, otherwise doesn't do anything.
+func EraseLine(w io.Writer) {
```

Comment:
hmm I think that's a good idea :+1:, the reason why I wrote it this way was because `Render(io.Writer)` takes as input writers that don't need to be files. But instead, I'll move this check inside of `Render`:
```go
if fw, ok := w.(FileWriter); ok {
     cursor.EraseLine(fw)
}
```

Reply:
Done!
I ended changing the `prog.Rende` function to accept the following signature:
```
Render(ctx context.Context, out FileWriteFlusher, r Renderer) error
```
which ended up being cleaner for erasing content.